NuGet.Docs/Create/Versioning.md

```diff
@@ -24,7 +24,7 @@ how to specify version ranges.
     (1.0,) = 1.0 < x
     (1.0,2.0) = 1.0 < x < 2.0
     [1.0,2.0] = 1.0 ≤ x ≤ 2.0
-    empty = latest version.
+    empty = lowest version.
```

Comment:
This should specify the behavior per version (< 2.8 latest, > 2.8 lowest), and recommend not to use it at all